week-1-practise/level-1/9-function-output-2.js

```diff
@@ -1,14 +1,14 @@
 function s(w1, w2) {
-    return w1.concat(w2);
+  return w1.concat(w2);
 }
 
-var result = s('code', 'your');
+var result = s("code", "your");
 console.log(result);
 
 // Answer these questions:
-// - What does this program do? Run it and see, or Google some of the methods used ot understand them.
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
-// - What does the function return?
-// - How can this function be improved?
\ No newline at end of file
+// - What does this program do? Run it and see, or Google some of the methods used ot understand them."concatenate two words together"
+// - How many parameters does the function take? 2 parameters
+// - What is the function name? s
+// - Where is the function called? with what parameters? line 5 with parameters "code","your"
+// - What does the function return? "concatenate string"
```

Comment:
I think a clearer description could be "a string formed of concatenating two of the function string parameters"